### DISCUSSION TITLE
Possibility to Specify .vercel folder or Project-Id with CLI for local monorepos.

### DISCUSSION THREAD

**Discussion by L-U-C-K-Y** [vercel/vercel#8922] [Ideas] [answered]
Dear all

We are using a self-hosted GitLab account and deploy to Vercel with our monorepo.
Git is not connected and we use the command `vercel deploy`.

So far, we just had to deploy one of the packages to Vercel, but now they are two packages.

In the .vercel folder, there is only one project id specified.

It would great if we could specify .vercel folder (we could create multiple) or the project-id where to deploy to.

Thanks!

**Comment by amyegan** [comment, marked as answer]
Thanks for sharing this idea @L-U-C-K-Y! 

In case this is a problem that has you stuck, I'm sharing the current recommendation. The existing process for [using local monorepos with Vercel CLI](https://vercel.com/docs/concepts/monorepos#using-monorepos-with-vercel-cli) involves deploying the projects one at a time. You would use your normal steps to deploy the first project, use [`vercel link`](https://vercel.com/docs/cli/link#) to switch to the second, and then deploy the second project.
